iceberg/iceberg-handler/src/test/queries/positive/mv_iceberg_orc8.q

```diff
@@ -0,0 +1,22 @@
+-- MV source table has varchar column.
```

Comment:
this qtest passes me on master without the fix, I'm running:
```
mvn install -Dtest.output.overwrite=true -Pitests,iceberg -Denforcer.skip=true -pl itests/qtest-iceberg -Dtest=TestIcebergLlapLocalCliDriver -Dqfile=mv_iceberg_orc8.q
```
how does this qtest validates the fix?

Reply:
The base table must be native and the MV must be Iceberg. I updated the test.

Reply:
okay, it fails now with NPE, thanks, resolving this conversation